openpype/hosts/blender/plugins/publish/validate_setdress_object_names.py

```diff
@@ -0,0 +1,40 @@
+from typing import List
+
+import bpy
+import string
+
+import pyblish.api
+
+import openpype.hosts.blender.api.action
+from openpype.pipeline.publish import ValidateContentsOrder
+
+
+class ValidateSetdressObjectNames(pyblish.api.InstancePlugin):
```

Comment:
Isn't `pyblish.api.Validator` better use here?

Reply:
I used an existing validator script from kaamaurice to create mine so I kept `pyblish.api.InstancePlugin`
 
But I'll test with `pyblish.api.Validator` if it works fine ! :)